sklearn/neighbors/tests/test_sample_weight.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pytest
+import scipy.stats
+
+from sklearn import neighbors
+from sklearn.utils._testing import assert_array_almost_equal
+
+
+@pytest.mark.parametrize("weights", ["uniform", "distance", scipy.stats.norm.pdf])
+def test_KNeighborsRegressor_sample_weight_none_is_same_as_uniform(
+    weights,
+    n_samples=40,
+    n_features=5,
+    n_test_pts=10,
+    n_neighbors=3,
+    random_state=0,
+):
```

Comment:
You can add a docstring to explain what the function is testing.

Reply:
ok